packages/main/src/themes/ToolbarPopover.css

```diff
@@ -0,0 +1,40 @@
+.ui5-overflow-list {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	padding: 0 0.5rem;
+}
+
+.ui5-overflow-list--alignleft .ui5-tb-popover-button::part(button) {
```

Comment:
Can we move these styles (`ui5-tb-popover-button::part(button)`) to dedicated `ToolbarButton.css` file? Or if they should be apply to all items inside the toolbar we can use the `ui5-tb-popover-item` class